src/Block/Service/ContainerBlockService.php

```diff
@@ -61,7 +61,11 @@ public function configureEditForm(FormMapper $form, BlockInterface $block): void
             'translation_domain' => 'SonataBlockBundle',
         ]);
 
-        $form->add('children', CollectionType::class);
+        $form->add('children', CollectionType::class, [], [
```

Comment:
This is needed so the collection renders as inline and starts working:
![image](https://user-images.githubusercontent.com/1137485/184810942-095edc5c-1a28-43f2-bd6e-f67d196da608.png)

Reply:
Reverted , this should be applied on SonataPageBundle.